### ISSUE TITLE
'java.lang.String com.activeandroid.TableInfo.getTableName()' on a null object reference

### ISSUE THREAD

**Issue by runat3** [pardom-zz/ActiveAndroid#543]
With this code, the problem persists.
```
<meta-data
    android:name="AA_MODELS"
    android:value="com.myapp.model.Item, com.myapp.model.Category" />
```
Then I found that by removing the space, it worked
like this
```
<meta-data
    android:name="AA_MODELS"
    android:value="com.myapp.model.Item,[No space here]com.myapp.model.Category" />
```

**Comment by ImangazalievM** [comment]
Hello. ActiveAndroid no longer be maintained. You can use [ReActiveAndroid](https://github.com/ImangazalievM/ReActiveAndroid), it's based on ActiveAndroid.

[Full Documentation](https://imangazalievm.gitbooks.io/reactiveandroid).

**Comment by WahidNasri** [comment]
try to disable instant run

**Comment by deepankar-np** [comment]
> try to disable instant run

Yes, It is working after disabling instant run. I also invalidated and restarted the android studio. 
However, after reinstalling the APKit was again throwing the same error. Hence, I cleared the app data and after that, the app again started working.

**Comment by ReejeshPK** [comment]
> try to disable instant run

That really helped. Thanks!
